### PR TITLE
add some reasons why

### PR DESCRIPTION
**Pull request by stanleygomes**
# PR Details

## Description

Give a good explanation about this PR, including about the files you've changed.

## Related Issue 

Issue Title ([PR_ID](http://github.com/link_for_issue))

## Steps to Reproduce

`Write your steps here:`

1. Sample
2. Of
3. Steps

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,29 @@ These are some of patterns definitions to help us to keep a default configuratio
...
+- Transform request and response data
+- Cancel requests
+- Automatic transforms for JSON data
+- Client side support for protecting against XSRF
+
+Why Material?
+
+- dynamic styles generated at runtime
+- nested themes with intuitive overrides
+- reduced load time with code splitting
```
Comment (hugonasciutti) [inline README.md]: Iniciar com letras maiúsculas para mantermos o padrão.

```suggestion
- Reduced load time with code splitting
```

Comment (stanleygomes) [reply README.md]: Ajustado.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,29 @@ These are some of patterns definitions to help us to keep a default configuratio
 - Linter: If You don't use [Gandalf Lint](https://github.com/SoftboxLab/gandalf-lint), you shall not pass.
 - Do you want a component pattern? [Get here](src/containers/Home/index.jsx)
 
+Why Formik?
```
Comment (vilaboim) [inline README.md]: I believe we can summarize this using links

```suggestion
[Why Formik?](https://jaredpalmer.com/formik/docs/overview#motivation)
```
I suggest these links:
[Why Formik?](https://jaredpalmer.com/formik/docs/overview#motivation)
[Why Axios?](https://github.com/axios/axios#features)
[Why Material-UI?](https://material-ui.com/blog/material-ui-v4-is-out/)

File: README.md [hunk 3]
```diff
@@ -41,6 +41,12 @@ These are some of patterns definitions to help us to keep a default configuratio
 - Linter: If You don't use [Gandalf Lint](https://github.com/SoftboxLab/gandalf-lint), you shall not pass.
 - Do you want a component pattern? [Get here](src/containers/Home/index.jsx)
 
+[Why Formik?](https://jaredpalmer.com/formik/docs/overview#motivation)
```
Comment (vilaboim) [inline README.md]: Coloque os itens na lista acima.

Comment (stanleygomes) [reply README.md]: @vilaboim seria isso?

**Pull request merged**